gm4_blossoming_pots/beet.yaml

```diff
@@ -0,0 +1,41 @@
+id: gm4_blossoming_pots
+name: Blossoming Pots
+version: 3.0.X
+  # should I start the version number over for gm4 release?
+data_pack:
+  load: .
+  overlays:
+    - formats:
+        min_inclusive: 61
+        max_inclusive: 61
+      directory: since_61
+
+resource_pack:
+  load: .
+
+pipeline:
+  - gm4.plugins.extend.module
+
+meta:
+  gm4:
+    versioning:
+      schedule_loops: [main]
+    website:
+      description: Decorated Pots and Flower Pots extended to allow all kinds of things to be placed in them! Decorate in Style!
+      #recommended:
+        #- 
+      notes:
+        - <add notes on version or incompatibilities?>
+    #modrinth: 
+      #project_id: null
+    #smithed:
+      #pack_id: null
+    planetminecraft: 
+      uid: null
+    video: null
+    wiki: null
+    credits:
+      Creator:
+        - runcows
```

Comment:
Don't forget to add yourself to `gm4/CONTRIBUTORS.json`. There you can add links to your social media accounts.